app/services/rss_reader/assembler.rb

```diff
@@ -32,7 +32,9 @@ def assemble
     private
 
     def get_tags
-      @categories.first(4).map { |tag| tag.lstrip[0..19] }.join(",")
+      @categories.first(4).map do |tag|
+        tag.delete(" ").gsub(/[^[:alnum:]]/i, "")[0..19]
+      end.join(",")
```

Comment:
I have to make an exception in RSS parsing because feed may contain tags with hyphens. Rather than flat out rejecting those feed import because of malformed tag, this will follow what v1 editor used to do.